app/js/components/overview/widgets/maintenancePlanLite/maintenancePlanLite.jade

```diff
@@ -5,13 +5,14 @@ section.widget-block
     span.light(translate) Summary
     a.cta-link(ui-sref='app.maintenance', translate) View planner
   .maintenance-plan-lite
-    .row.row-short.padding-top(ng-if='getFirstSuggestion()')
+    //- .row.row-short.padding-top(ng-if='getFirstSuggestion()')
+    .row.row-short.padding-top
```

Comment:
ng-if='getFirstSuggestion()' should be here. Account 540155 shows a 0 action suggestion.